bundles/binding/org.openhab.binding.zwave/database/aeon/zw096.xml

```diff
@@ -205,7 +204,6 @@
             <Type>byte</Type>
             <Default>505050</Default>
```

Comment:
This value should be in decimal. I know you didn't add it, but do you mind fixing it?